pytket/extensions/qiskit/qiskit_convert.py

```diff
@@ -755,6 +735,7 @@ def append_tk_command_to_qiskit(
     cregmap: dict[str, ClassicalRegister],
     symb_map: dict[Parameter, sympy.Symbol],
     range_preds: dict[Bit, tuple[list["UnitID"], int]],
+    condition: tuple[ClassicalRegister | Clbit, int] | None = None,
 ) -> InstructionSet:
```

Comment:
TODO check that this return type of `InstructionSet` is still correct. I think it just returns a `QuantumCircuit` now.

Reply:
mypy doesn't seem to complain?